packages/starlight/components/Select.astro

```diff
@@ -30,7 +30,8 @@ interface Props {
 <style>
 	label {
 		--sl-label-icon-size: 0.875rem;
-		--sl-caret-size: 1.25rem;
+		--sl-caret-size: 1rem;
```

Comment:
Thanks @evadecker!

One detail here: modifying this variable also impacted the actual size of the caret icon on the right of the drop down menu, shrinking them ever so slightly

| Before | After |
|---|---|
| ![starlight astro build_](https://github.com/withastro/starlight/assets/357379/3816820e-117f-43d0-a3c7-4d3d833fab19) | ![starlight-git-fork-evadecker-dropdown-padding-astrodotbuild vercel app_](https://github.com/withastro/starlight/assets/357379/1a4bf222-e944-4a72-a8f4-922211132434) |

Would be nice for the solution not to reduce that if possible!

Reply:
That was intentional on my part to get the stroke width of the icons to match more closely. :) But if it's undesired I'm happy to revert to the existing size.

Reply:
Ah! Got it. Just cross referenced the [original design spec in Figma](https://www.figma.com/design/GrDkcguAR7tSWPFuLVDixq/Theme---Starlight-Docs?node-id=171-456&t=PwKzrekoyiytRa4V-4), which does have the size we’re currently using. 

I think it might be safest for this PR to just fix the outline positioning and tackle icon size later — for example this change makes the size inconsistent with the same icon in the left sidebar and in `<details>` styling. To some extent that makes sense given the context, font size variation etc., but probably something to chat about so wouldn’t want to block the fix on that.

I _would_ love to hear general feedback you might have about design details like this though — maybe something to tackle a bit more holistically if you’re interested?

Reply:
That makes sense! Reverted the icon size for now and would be happy to contribute some design polish feedback separately.